makePDF.py

```diff
@@ -379,6 +383,12 @@ def run ( self ):
 
 			self.caller.output(content)
 			self.caller.output("\n\n[Done!]\n")
+
+			if _HAS_PHANTOMS:

```

Comment:
I will just use `self.caller.errors = locals().get("errors", [])` and additionally define them as class variables to ensure, that they are always available.